test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenCreateSpecs.java

```diff
@@ -65,6 +66,8 @@ public static void main(String... args) {
 	@Override
 	protected List<HapiApiSpec> getSpecsInSuite() {
```

Comment:
Once `getInfo` is implemented, we should add tests regarding `TokenSupply` - `INIFINITE` and `FINITE`